README.md

```diff
@@ -1 +1,156 @@
-# country-global
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+   - [Usage](#usage)
+- [👥 Authors](#authors)
+ - [🚀 Live Demo](#-live-demo-)
```

Comment:
- [ ] This live demo section link is broken. Kindly ensure it works and leads to the live demo section.